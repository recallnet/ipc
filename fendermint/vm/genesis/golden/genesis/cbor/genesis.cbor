a96a636861696e5f6e616d65657cc2a7672b6974696d657374616d70006f6e6574776f726b5f76657273696f6e1568626173655f66656551008e89061654fdfb3555b2ecbbc6dde4c66b706f7765725f7363616c65206a76616c696461746f72738aa36a7075626c69635f6b6579584104d3f8641827bc9aa5ef801567dcd24950f879f9411a57f63703e14a93ec446f919fd330dd8383e1f06c13220eeb719d437ee1a233677cfd184ac8adfe2e137b9065706f776572490020d5d6640e85af886e73746f726167655f616d6f756e741bffffffffffffffffa36a7075626c69635f6b657958410435144adb4d9866fc60ce27356ae05714d7970a83dbb85a1af99b85ca79ffc6b62a69094fbf0aa8ca4ef6e6850c87495ef84f11f73aa5d7a1de3745098941b6e365706f776572510019210cfbc70d85957523125f59a4c41e6e73746f726167655f616d6f756e741b428412ef67333ef8a36a7075626c69635f6b6579584104563f83318a322a4851d23fc5e28c413c78ae2f47914f159427316ed71b64132bf23972119b6f77b0eb2119c379418e17ef7e9942091e082a78dac0b1a3b243ae65706f776572510025da4796ea8653387e1cb445e2f88c1b6e73746f726167655f616d6f756e741b933b04e1f68179b7a36a7075626c69635f6b65795841043f1b8f4cab2a0be16e6f6fb167941b6a7404df5a4cd953a08ec9729be11f95c93eb816876ce79f674d86b8382dd327f4797b431ed131742893eb25e3747d039765706f77657251003c5f4408ce7a4ef3afe1b296bc12d2d56e73746f726167655f616d6f756e741b304a3e689fb12336a36a7075626c69635f6b657958410496d542bce9fd49e3b8e22b220fc83243c1ce4d653a2cae3ea6d4d93d5a08955feb1c3ae03aa521c819ad50c276949073b4300860734f8cb6b0cde7080c2f75c065706f7765725100ef07784b70db874a64b3183fdf7a97246e73746f726167655f616d6f756e741be3546fedd7daa0e3a36a7075626c69635f6b65795841040d9ae8c95a932cd83011ae2f1132db240c1005492a921ba98c742af8204d25f88afdbdff58c43c8a584e8cb9817ebf4aa06894555e73c09d567deebab6669f6865706f7765725100d6ab9d6cd3c9596e78d11a5d9fd653c76e73746f726167655f616d6f756e741b46560cf732b9e942a36a7075626c69635f6b65795841048922d573f264942908e1eff39347d75db5758d9022b7669fd0185eaacc5d71fe7a8d791d6b8fd0ae0b25f15c82fb20e4eec42f0f76e4dbcbefe443292c41177b65706f7765725100a65b058c4e6cb547a9b2fc0b6f6e68516e73746f726167655f616d6f756e741b4f448663a59a867ba36a7075626c69635f6b65795841047b33a2b007a8e909fd567a3ddedf5483fe0b37eddf2880ecb38c97d1ed837667931fef4e24d11703d806727afde83b44b3c512fb5b6598437a003a8434740e2665706f7765725100ed9e94e5ec021f4df076ff26dbb725726e73746f726167655f616d6f756e7400a36a7075626c69635f6b6579584104e136ec009a4dc22fca807e023bd5615828619934ed4f81a3352c463c1a52fbd1bc4c79c7c7f93920300699b333bfca5cc033d3bc95fa0c202c33167065d369b365706f7765725100bf20d4428b2b431e96cd32ddcf0b2d9f6e73746f726167655f616d6f756e741bde46322dbb21b49aa36a7075626c69635f6b657958410434c01229fabbe01929fe1bfc8ae6f3c14b6ba407cb12956c7367c5d8507a68776a568d44f79902705b7c80bd0d9dc6c0afaf4eeacda648e25755807320bdbdea65706f7765725100c70bffc684a439119141f7926dd588a06e73746f726167655f616d6f756e741bbd392ac30a97e61e686163636f756e747384a2646d657461a1684d756c7469736967a4677369676e65727385550193f2c79630afb4b4cbc7ce8effd7102bf811086b55014c8daccb7d757f5949342128a01f42560a1d4bc3550191a94054ca4800e5ae98472b24597483617e305e5501ffc3a282ee1e5614fea44ba5227d2c5b43659a3f550166a7c82874e72334bd2119d2294fdd67b3d5670a697468726573686f6c64057076657374696e675f6475726174696f6e1b1519ac1b0e1169636d76657374696e675f73746172741b1a4f91c2e2d24b916762616c616e636540a2646d657461a1674163636f756e74a1656f776e65725501b0680710a3150456ae76f8d74f64efb264354bb76762616c616e63655100e911e498583207bf722f2dbb4f96ffcca2646d657461a1674163636f756e74a1656f776e6572550116c8372a93ade717d37403cac0a93d9e82f7419d6762616c616e6365510017903d4a9ac5d334f0de4a7f4591f98ba2646d657461a1674163636f756e74a1656f776e657255011814a706b88f7962ab66c6e96b3a667a6277df296762616c616e6365510059eb83f30c730291f3720f3404b35ba57365616d5f7065726d697373696f6e5f6d6f6465a1646d6f64656c756e7265737472696374656463697063a16767617465776179a4697375626e65745f6964821b40d59e1552a655af8076626f74746f6d5f75705f636865636b5f706572696f641b010a528ad819d814736d616a6f726974795f70657263656e74616765184e776163746976655f76616c696461746f72735f6c696d69741840