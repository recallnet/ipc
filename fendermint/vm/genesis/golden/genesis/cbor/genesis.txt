Genesis { chain_name: "|Â§g+", timestamp: Timestamp(0), network_version: NetworkVersion(21), base_fee: TokenAmount(189461843534858307916.482632770341692614), power_scale: -1, validators: [Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [4485009, 5416187, 56848446, 17066335, 16284153, 53776980, 87677, 40540094, 1583036, 3472921], magnitude: 1, normalized: true }, y: Field { n: [34831248, 36405131, 64062636, 13475315, 58646946, 47998800, 20062446, 59228592, 14517123, 2618572], magnitude: 1, normalized: true }, infinity: false })), power: Collateral(TokenAmount(2.366032904481779592)), storage_amount: StorageAmount(18446744073709551615) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [33539766, 48329374, 27373465, 34565857, 14128906, 45618629, 14840662, 26997123, 47926680, 869650], magnitude: 1, normalized: true }, y: Field { n: [21083875, 30491234, 24780259, 64809623, 49827601, 52548183, 57567312, 44247355, 22003466, 694850], magnitude: 1, normalized: true }, infinity: false })), power: Collateral(TokenAmount(33402309027742444856.580864060816671774)), storage_amount: StorageAmount(4792976722866945784) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [56890155, 6010310, 22626931, 18761020, 7908911, 10686543, 19135582, 11084103, 53578290, 1413088], magnitude: 1, normalized: true }, y: Field { n: [62014382, 45100136, 8562573, 17310840, 66027161, 38822789, 34708535, 31376300, 34708335, 3968604], magnitude: 1, normalized: true }, infinity: false })), power: Collateral(TokenAmount(50314808567090597485.257437776254110747)), storage_amount: StorageAmount(10609078715751168439) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [18847177, 39626488, 20580588, 23671653, 41157855, 31786714, 49740566, 3114425, 55356202, 1033955], magnitude: 1, normalized: true }, y: Field { n: [8192919, 46758109, 54692158, 8078533, 7961411, 57985533, 7046018, 41786678, 42429671, 1027589], magnitude: 1, normalized: true }, infinity: false })), power: Collateral(TokenAmount(80248327850235883239.650499776769872597)), storage_amount: StorageAmount(3479662281173836598) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [34116959, 20336470, 48491117, 26536114, 63032909, 66194576, 35828256, 19369699, 45935101, 2471248], magnitude: 1, normalized: true }, y: Field { n: [3110336, 58311171, 13331212, 25283902, 62140424, 27599900, 47516711, 8855654, 48249509, 3852046], magnitude: 1, normalized: true }, infinity: false })), power: Collateral(TokenAmount(317724276936796978933.962464497539651364)), storage_amount: StorageAmount(16380840811972698339) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [5055992, 17481224, 29006023, 19180104, 790533, 5027529, 18539249, 11755712, 13195923, 222906], magnitude: 1, normalized: true }, y: Field { n: [40279912, 58437293, 644455, 22378959, 44066964, 6270930, 15256472, 15870305, 33511620, 2277231], magnitude: 1, normalized: true }, infinity: false })), power: Collateral(TokenAmount(285345866821275530669.209387609108140999)), storage_amount: StorageAmount(5068252686499572034) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [6124030, 35105459, 40500481, 37784285, 28669325, 13759959, 35585849, 38839331, 24375908, 2246837], magnitude: 1, normalized: true }, y: Field { n: [4265851, 17877579, 29146878, 4053907, 15647791, 12503097, 39785928, 54704172, 18705295, 2007902], magnitude: 1, normalized: true }, infinity: false })), power: Collateral(TokenAmount(221124458842684725296.451267019777402961)), storage_amount: StorageAmount(5711837989950883451) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [25392743, 52819067, 969528, 62356642, 66980663, 62379296, 23569373, 61089781, 45090728, 2018536], magnitude: 1, normalized: true }, y: Field { n: [7605798, 958733, 25442208, 65891734, 11781394, 58330833, 6760367, 6033248, 55452881, 2410491], magnitude: 1, normalized: true }, infinity: false })), power: Collateral(TokenAmount(315850437917585148857.830390091962852722)), storage_amount: StorageAmount(0) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [38992849, 51482374, 1717074, 13874494, 2646425, 49633366, 516131, 50904874, 39501, 3689915], magnitude: 1, normalized: true }, y: Field { n: [30632371, 12950553, 12714691, 49436648, 12596179, 15725207, 6921011, 14975168, 29870073, 3085086], magnitude: 1, normalized: true }, infinity: false })), power: Collateral(TokenAmount(254053005837357463442.549747876184665503)), storage_amount: StorageAmount(16016544296784213146) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [8022135, 32601620, 22464310, 2042954, 21719972, 45726960, 65126344, 58746023, 36305595, 864260], magnitude: 1, normalized: true }, y: Field { n: [12434922, 23076040, 9315701, 61552281, 11513678, 57110960, 63441872, 639341, 21297049, 1742243], magnitude: 1, normalized: true }, infinity: false })), power: Collateral(TokenAmount(264578674169317805441.853830822618302624)), storage_amount: StorageAmount(13634976364025603614) }], accounts: [Actor { meta: Multisig(Multisig { signers: [SignerAddr(Address("f1spzmpfrqv62ljs6hz2hp7vyqfp4bccdl4lujoqi")), SignerAddr(Address("f1jsg2zs35ov7vssjueeukah2ckyfb2s6dtvbatza")), SignerAddr(Address("f1sguuavgkjaaolluyi4vsiwluqnqx4mc6a5alwby")), SignerAddr(Address("f177b2faxodzlbj7vejosse7jmlnbwlgr7mka5lri")), SignerAddr(Address("f1m2t4qkdu44rtjpjbdhjcst65m6z5kzyksecek6i"))], threshold: 5, vesting_duration: 1520435581414369635, vesting_start: 1895894234361383825 }), balance: TokenAmount(0.0) }, Actor { meta: Account(Account { owner: SignerAddr(Address("f1wbuaoefdcucfnltw7dlu6zhpwjsdks5xzjmyxyy")) }), balance: TokenAmount(309803028523845988948.688478674194202572) }, Actor { meta: Account(Account { owner: SignerAddr(Address("f1c3edokutvxtrpu3uapfmbkj5t2bpoqm5qwx5tji")) }), balance: TokenAmount(31321177788451776357.618988345927530891) }, Actor { meta: Account(Account { owner: SignerAddr(Address("f1dakkobvyr54wfk3gy3uwwotgpjrhpxzjjnpxeeq")) }), balance: TokenAmount(119524157638567625240.367847145032866725) }], eam_permission_mode: Unrestricted, ipc: Some(IpcParams { gateway: GatewayParams { subnet_id: SubnetID { root: 4671814002884892079, children: [] }, bottom_up_check_period: 74963100089571348, majority_percentage: 78, active_validators_limit: 64 } }) }